service-base/src/test/java/org/eclipse/hono/service/metric/MicrometerBasedMetricsTest.java

```diff
@@ -218,4 +219,99 @@ public void testPayloadSizeForCommandMessages(final MeterRegistry registry) {
                 registry.find(MicrometerBasedMetrics.METER_COMMANDS_PAYLOAD).summary().totalAmount());
     }
 
+    /**
+     * Verifies that the metrics for a tenant are removed when all their connections are closed.
+     *
+     * @param registry The registry that the tests should be run against.
+     */
+    @ParameterizedTest
+    @MethodSource("registries")
+    public void testAuthenticatedConnections(final MeterRegistry registry) {
+
+        final MicrometerBasedMetrics metrics = new MicrometerBasedMetrics(registry);
+
+        final String tenant1 = "tenant1";
+        final String tenant2 = "tenant2";
+
+        // GIVEN a metrics instance that has recorded one telemetry message and one event per tenant
```

Comment:
IMHO this comment does not reflect what we are setting up as the fixture, does it?